VultisigApp/VultisigApp/Services/Thorchain/ThorchainService.swift

```diff
@@ -75,4 +77,24 @@ class ThorchainService {
         
         return cacheEntry.balances
     }
+    
+    func fetchFeePrice() async throws -> UInt64 {
+        let cacheKey = "thorchain-fee-price"
+        if let cachedData: UInt64 = await Utils.getCachedData(cacheKey: cacheKey, cache: cacheFeePrice, timeInSeconds: 60*5) {
```

Comment:
we can use URLRequest cache for that kind of things. 
https://developer.apple.com/documentation/foundation/nsurlrequest/cachepolicy